debian/postinst

```diff
@@ -52,30 +52,10 @@ if [ $(cat /etc/passwd | grep ^openquake: | wc -l) -eq 0 ]; then
     adduser --system --home /var/lib/$USER --group --shell /bin/bash $USER
 fi
 
-HDIR=/usr/share/openquake/engine
-IDIR=/usr/lib/python2.7/dist-packages/openquake/engine
-mkdir -p $HDIR
-
-rm -f $HDIR/celeryconfig.py.new_in_this_release
-cp $IDIR/celeryconfig.py $HDIR
+HDIR=/usr/share/openquake
+mkdir -p ${HDIR}/engine
 chown -R $USER.$USER $HDIR
```

Comment:
With HDIR=/usr/share/openquake you are changing things that potentially are managed by other packages, why this change ?